store/postgres/src/store.rs

```diff
@@ -1110,6 +1111,15 @@ impl SubgraphDeploymentStore for Store {
         Ok(self.cached_schema(subgraph_id)?.api)
     }
 
+    fn fulltext_fields(
+        &self,
+        subgraph_id: &SubgraphDeploymentId,
+    ) -> Result<SubgraphFulltextEntities, Error> {
+        Ok(Schema::subgraph_fulltext_entity_fields(
```

Comment:
The naming is tricky here because the result is a nested map of entity name -> that entities fulltext included fields -> the fulltext query field that the included field contributes to. 

You'll see the SubgraphFulltextEntities map has an inner type SubgraphFulltextFields, so ultimately this data is used to map an entity to fulltext fields hence the method name.

Reply:
This is going away anyway, as I'm moving the fulltext_entities data to the Layout.